edx_proctoring/templates/proctored_exam/pending-prerequisites.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+<div class="warning sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      This exam is proctored.
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+    This exam is proctored but you have not completed all of the prerequisites for this exam. Either you have not yet attempted these prerequisites or the results of your attempts are still pending. All requirements for this exam must be passed before you can proceed to take this proctored exam. You can see the requirements listed (in order) on your <a href="{{progress_page_url}}">Progress</a> page.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+    The following prerequisites are considered <strong>pending</strong> and must be completed (and successful) before proceeding:
+    {% endblocktrans %}
+    <ol style='list-style-type: disc'>
+    {% for requirement in prerequisite_status.pending_prerequisites %}
+      <li>
+        {% if requirement.jumpto_url %}
+          <a href='{{requirement.jumpto_url}}'>{{requirement.display_name}}</a>
+        {% else %}
+          {{requirement.display_name}}
+        {% endif %}
+      </li>
+    {% endfor %}
+    </ol>
+  </p>
+  <p>
+    {% blocktrans %}
+    Please confirm that you have completed theses requirements. Or you can decide to take this exam without proctoring which will make you ineligible for credit.
```

Comment:
Not really. I separated out the pending versus failing templates just because it seems like messaging needs to be different. Basically, the actions are:

- decline proctoring

- if you haven't attempted pre-requisites, go do them

or

- if you have attempted the pre-requisites and are waiting for results, please come back later and hopefully the submitted requirements have been processed